daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/value/Value.scala

```diff
@@ -194,15 +197,21 @@ object Value {
   type VersionedContractInstance = transaction.Versioned[ContractInstance]
 
   object VersionedContractInstance {
-    def apply(template: Identifier, arg: VersionedValue): VersionedContractInstance =
-      arg.map(ContractInstance(template, _))
+    def apply(
+        packageName: Option[Ref.PackageName] = None,
```

Comment:
Is the plan to eventually remove the default value? If not then maybe we should also add TODOs to the use sites in Canton or we may forget to update them. Or will there be cases where it makes sense not to provide a package name?

Reply:
add a todo.